### ISSUE TITLE
analyzer 0.40.x support

### ISSUE THREAD

**Issue by proninyaroslav** [dart-lang/mockito#289]
Unfortunately I can't use `analyzer 0.40.x`:
```
Because mockito >=4.1.2 depends on analyzer ^0.39.15 and ... depends on analyzer ^0.40.4, mockito >=4.1.2 is forbidden.

So, because ... depends on mockito ^4.1.2, version solving failed.
pub get failed (1; So, because ... depends on mockito ^4.1.2, version solving failed.)
exit code 1
```

**Comment by srawlins** [comment]
I'll release a new version ASAP which allows analyzer 0.40.

**Comment by shinayser** [comment]
> I'll release a new version ASAP which allows analyzer 0.40.

Thanks @srawlins . There is a ETA when we getting this new version? I cant run tests with hivedb right now because of mockito using old version of analyzer =(

**Comment by srawlins** [comment]
No ETA.

You should be able to override the analyzer dependency to 0.39 via [dependency_overrides](https://dart.dev/tools/pub/pubspec#dependencies).

**Comment by srawlins** [comment]
Blocked by #291

**Comment by srawlins** [comment]
Fixed in 032b4d157f8719f43172c5e704e77cf3ac5bd42a

**Comment by srawlins** [comment]
mockito 4.1.3 has support for analyzer 0.40.
